client/config/config.go

```diff
@@ -0,0 +1,152 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	tmcli "github.com/tendermint/tendermint/libs/cli"
+)
+
+var ErrWrongNumberOfArgs = fmt.Errorf("wrong number of arguments")
+
+type ClientConfig struct {
+	ChainID        string `mapstructure:"chain-id"`
+	KeyringBackend string `mapstructure:"keyring-backend"`
+	Output         string `mapstructure:"output"`
+	Node           string `mapstructure:"node"`
+	BroadcastMode  string `mapstructure:"broadcast-mode"`
+	Trace          bool   `mapstructure:"trace"`
+}
+
+func DefaultClientConfig() *ClientConfig {
+	return &ClientConfig{
+		ChainID:        "",
+		KeyringBackend: "os",
+		Output:         "text",
+		Node:           "tcp://localhost:26657",
+		BroadcastMode:  "sync",
+		Trace:          false,
```

Comment:
we shouldn't hardcode these strings, but should put them into consts (or use existing ones if they exist)

Reply:
Done!